engine/src/main/java/org/terasology/rendering/primitives/BlockMeshGeneratorSingleShape.java

```diff
@@ -149,7 +150,7 @@ private boolean isSideVisibleForBlockTypes(Block blockToCheck, Block currentBloc
             return true;
         }
 
-        if (blockToCheck.getURI().toString().equals("engine:unloaded")) {
+        if (blockToCheck.getURI().equals(BlockManager.UNLOADED_ID)) {
```

Comment:
comparing against a constant is way better anyways. and if this ends up doing less work that's even better! :+1: